components/ScrollingAvatars/ScrollingAvatarsContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC, useState } from 'react';
+import _ from 'underscore';
+
+import ScrollingAvatars from './ScrollingAvatars';
+
+interface AvatarInfo {
+  name: string;
+  position: string;
+  imgSrc: string;
+  key?: number;
```

Comment:
not sure we even need key here. I think using a unique string identifier is good and we have that in `name`. Plus, it saves on some logic below if we just key by `name`

it isn't exactly the same pattern but I would read [this](https://medium.com/geekculture/reactjs-why-index-as-a-key-is-an-anti-pattern-4b9dc6ef0067)

Reply:
hmm I'll just remove it then